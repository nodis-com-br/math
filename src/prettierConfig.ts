export const prettierConfig = {
  endOfLine: 'lf',
  semi: true,
  singleQuote: true,
  tabWidth: 2,
  trailingComma: 'all',
  overrides: [
    {
      files: '*.json',
      options: {
        printWidth: 200,
      },
    },
    {
      files: '*.scss',
      options: {
        singleQuote: true,
        printWidth: 200,
      },
    },
  ],
};
