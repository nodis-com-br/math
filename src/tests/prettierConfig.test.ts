import { prettierConfig } from '../prettierConfig';

describe('prettierConfig', () => {
  test('should match snapshot', () => {
    expect(prettierConfig).toMatchSnapshot();
  });
});
