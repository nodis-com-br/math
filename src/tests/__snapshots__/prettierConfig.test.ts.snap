// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prettierConfig should match snapshot 1`] = `
Object {
  "endOfLine": "lf",
  "overrides": Array [
    Object {
      "files": "*.json",
      "options": Object {
        "printWidth": 200,
      },
    },
    Object {
      "files": "*.scss",
      "options": Object {
        "printWidth": 200,
        "singleQuote": true,
      },
    },
  ],
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
}
`;
